.github/workflows/auto-build.yml

```diff
@@ -41,6 +41,8 @@ jobs:
           sudo chmod +x amd64/bin/kube* && sudo chmod +x arm64/bin/kube*
           sudo wget https://sealer.oss-cn-beijing.aliyuncs.com/sealer-latest.tar.gz && sudo tar -xvf sealer-latest.tar.gz -C /usr/bin
           sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
+          sudo sed -i "s/v1.19.8/$version/g" amd64/Metadata
```

Comment:
one PR only does one thing, pls fire those github action changes to another one.